corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -655,7 +653,13 @@ def sync_log(self):
 
     @property
     def async_cache_key(self):
-        return restore_cache_key(self.domain, ASYNC_RESTORE_CACHE_KEY_PREFIX, self.restore_user.user_id)
+        return restore_cache_key(
```

Comment:
i thought about doing this as well but held off because it changed this: https://github.com/dimagi/commcare-hq/blob/br/replicate-couch/corehq/form_processor/submission_post.py#L228 (the invalidation). i wasn't sure in what ways async restore needed that. i'm guessing it doesn't make much of a difference. in regular syncs it doesn't matter because it deletes the caches if there is no device id / sync log id and it's irrelevant if the cache has the sync log id since the cache won't get used again. i wasn't sure if i could make the same assumptions about async restore. guessing you can, but wanted to flag just in case